Client/Frontend/Widgets/TwoLineCell.swift

```diff
@@ -163,4 +175,34 @@ private class TwoLineCellHelper {
             detailTextLabel.text = detailText
         }
     }
+
+    func mergeAccessibilityLabels(labels: [AnyObject?]?) {
+        let labels = labels ?? [textLabel, imageView, detailTextLabel]
+
+        let label = labels.map({ (var label: AnyObject?) -> NSAttributedString? in
+            if let view = label as? UIView {
+                label = view.valueForKey("accessibilityLabel")
+            }
+
+            if let attrString = label as? NSAttributedString {
+                return attrString
+            } else if let string = label as? String {
+                return NSAttributedString(string: string)
+            } else {
+                return nil
+            }
+        }).filter({
+            $0 != nil
+        }).reduce(NSMutableAttributedString(string: ""), combine: {
+            if ($0.length > 0) {
+                $0.appendAttributedString(NSAttributedString(string: ", "))
+            }
+            $0.appendAttributedString($1!)
+            return $0
+        })
+
+        container.isAccessibilityElement = true
+        println("Setting label: \"\(label)\"")
```

Comment:
I love map/reduce/filter but I think a simpler for loop would also work here. But lets leave it.

Can you just remove the `println()` before I merge this?